doc/user/user_install.rst

```diff
@@ -6,12 +6,12 @@ This section will guide you through installing the ARMI Framework on your machin
 Prerequisites
 -------------
 These instructions target users with some software development knowledge. In
-particular, we assume familiarity with `Python <https://www.python.org/>`_,
+particular, we assume familiarity with `Python <https://www.python.org/>`__,
 `virtual environments <https://docs.python.org/3/tutorial/venv.html>`_, and `Git <https://git-scm.com/>`_.
 
 You must have the following installed before proceeding:
 
-* `Python <https://www.python.org/downloads/>`_ version 3.6 or later (preferably 64-bit)
+* `Python <https://www.python.org/downloads/>`__ version 3.6 or later (preferably 64-bit)
```

Comment:
this one has gotta be the silliest. But yeah duplicate labels so needed to use anonymous hyperlinks: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks

Reply:
Wow, yeah, that's pretty minor.  Definitely hadn't seen anon links in RST before. Well... learned something new!